GiEnJul/ClientApp/src/routes/registerGiver/SummaryRegistration.tsx

```diff
@@ -27,23 +30,92 @@ const SummaryRegistration: React.FC<Props> = ({ nextStep, prevStep, submit, valu
         prevStep();
     }
 
+    const classes = useStyles();
+
     return (
         <div>
-            <h5>Sted:</h5>
-            <h6>{values.location}</h6>
-            <h5>Navn:</h5>
-            <h6>{values.fullname}</h6>
-            <h5>Epostadresse:</h5>
-            <h6>{values.email}</h6>
-            <h5>Telefonnummer:</h5>
-            <h6>{values.phoneNumber}</h6>
-            <h5>Ønsket familiesammensetning:</h5>
-            <h6>{values.familyType}</h6>
-
-            <ButtonToolbar>
-                <Button onClick={Previous}>Tilbake</Button>
-                <Button onClick={Submit} >Bli giver</Button>
-            </ButtonToolbar>
+            <FormControl 
+          variant="outlined" 
+          fullWidth
+          required
+          margin = "normal"
+          style={{width: '100%', marginTop: '20px'}}>
```

Comment:
It would be nice to have this style in a style file as an id or class.

Reply:
True! If we change the class component to function component it is easy to to with hooks. I can make an issue on it.